test/batch/batching_test.go

```diff
@@ -45,85 +46,85 @@ func TestBatch_integration(t *testing.T) {
+	t.Run("POST /batch/references", func(t *testing.T) {
 		client := testsuit.CreateTestClient()
 		testsuit.CreateWeaviateTestSchemaFoodWithReferenceProperty(t, client)
 
 		// Create some objects
-		classT := &models.Thing{
+		classT := &models.Object{
 			Class: "Pizza",
 			ID:    "97fa5147-bdad-4d74-9a81-f8babc811b09",
-			Schema: map[string]string{
+			Properties: map[string]string{
 				"name":        "Doener",
 				"description": "A innovation, some say revolution, in the pizza industry.",
 			},
 		}
-		batchResultT, batchErrT := client.Batch().ThingsBatcher().WithObject(classT).Do(context.Background())
+		batchResultT, batchErrT := client.Batch().ObjectsBatcher().WithObject(classT).Do(context.Background())
 		assert.Nil(t, batchErrT)
 		assert.NotNil(t, batchResultT)
-		classA := &models.Action{
+		classA := &models.Object{
 			Class: "Soup",
 			ID:    "07473b34-0ab2-4120-882d-303d9e13f7af",
-			Schema: map[string]string{
+			Properties: map[string]string{
 				"name":        "Beautiful",
 				"description": "Putting the game of letter soups to a whole new level.",
 			},
 		}
-		batchResultA, batchErrA := client.Batch().ActionsBatcher().WithObject(classA).Do(context.Background())
+		batchResultA, batchErrA := client.Batch().ObjectsBatcher().WithObject(classA).Do(context.Background())
 		assert.Nil(t, batchErrA)
 		assert.NotNil(t, batchResultA)
 		time.Sleep(2.0 * time.Second)
 
 		// Define references
 		refTtoA := &models.BatchReference{
-			From: "weaviate://localhost/things/Pizza/97fa5147-bdad-4d74-9a81-f8babc811b09/otherFoods",
-			To:   "weaviate://localhost/actions/07473b34-0ab2-4120-882d-303d9e13f7af",
+			From: "weaviate://localhost/Pizza/97fa5147-bdad-4d74-9a81-f8babc811b09/otherFoods",
+			To:   "weaviate://localhost/07473b34-0ab2-4120-882d-303d9e13f7af",
 		}
 		refTtoT := client.Batch().ReferencePayloadBuilder().WithFromClassName("Pizza").WithFromRefProp("otherFoods").WithFromID("97fa5147-bdad-4d74-9a81-f8babc811b09").WithToID("97fa5147-bdad-4d74-9a81-f8babc811b09").Payload()
 
 		refAtoT := &models.BatchReference{
-			From: "weaviate://localhost/actions/Soup/07473b34-0ab2-4120-882d-303d9e13f7af/otherFoods",
-			To:   "weaviate://localhost/things/97fa5147-bdad-4d74-9a81-f8babc811b09",
+			From: "weaviate://localhost/Soup/07473b34-0ab2-4120-882d-303d9e13f7af/otherFoods",
+			To:   "weaviate://localhost/97fa5147-bdad-4d74-9a81-f8babc811b09",
 		}
-		refAtoA := client.Batch().ReferencePayloadBuilder().WithFromKind(semantics.Actions).WithFromClassName("Soup").WithFromRefProp("otherFoods").WithFromID("07473b34-0ab2-4120-882d-303d9e13f7af").WithToKind(semantics.Actions).WithToID("07473b34-0ab2-4120-882d-303d9e13f7af").Payload()
+		refAtoA := client.Batch().ReferencePayloadBuilder().WithFromKind(semantics.Objects).WithFromClassName("Soup").WithFromRefProp("otherFoods").WithFromID("07473b34-0ab2-4120-882d-303d9e13f7af").WithToKind(semantics.Objects).WithToID("07473b34-0ab2-4120-882d-303d9e13f7af").Payload()
```

Comment:
`.WithFromKind()` and `.WithToKind()` should be removed from ReferencePayloadBuilder

Reply:
done